web-server/assets/src/script/skills.js

```diff
@@ -65,6 +65,9 @@ const matchesFilters = (filters, string) => {
 }
 
 const filterSkillsList = filter => {
+  const queryString = filter === '' ? '' : `?f=${filter}`
```

Comment:
lets parse the whole querystring, modify it, serialize it, and then replaceState